lms/envs/bok_choy.py

```diff
@@ -76,6 +76,16 @@
 CELERY_ALWAYS_EAGER = True
 CELERY_RESULT_BACKEND = 'djcelery.backends.cache:CacheBackend'
 
+BLOCK_STRUCTURES_SETTINGS = dict(
+    # We have CELERY_ALWAYS_EAGER set to True, so there's no asynchronous
+    # code running and the celery routing is unimportant.
+    # It does not make sense to retry.
+    BLOCK_STRUCTURES_TASK_MAX_RETRIES=0,
+    # These delay values are irrelevent because we should never retry.
+    BLOCK_STRUCTURES_COURSE_PUBLISH_TASK_DELAY=0,
```

Comment:
Ah, thanks for the correction. I'll update the comment to be accurate.

As this wasn't the root cause of the bok-choy failures on master, I'll hold off merging until rebasing with the (as yet undetermined, but my theory is that it has to do with a change in Google's YouTube iFrame API) fix for that.